src/components/ContainerSection/index.tsx

```diff
@@ -80,8 +68,7 @@ const Section: React.FC<SectionProps> = ({
       <Carousel items={carousel} />
       <div className="container__button">
         <Button
-          link="https://srdentech.vercel.app/saiba-mais"
-          target="_blank"
+          link="https://srdentech.vercel.app/construcao"
```

Comment:
A prop `link` é a `to` do componente Link do react-router, então você pode passar a rota direto. Como ainda não tá recebendo a rota via props, por enquanto, pode ser assim:

`link="construcao"`

Se passar a URL completa, ele faz o redirecionamento pra produção, então a página toda recarrega.